test/test-classTetrisMoves.js

```diff
@@ -0,0 +1,301 @@
+            --------
+            --------
+        `)
+        assert.deepEqual(elementTetris.tetromino.coordinates, expectedNewTetrominoI)
+        elementTetris.moveRight()
+        elementTetris.tick()
+        elementTetris.tick()
+        elementTetris.tick()
+        elementTetris.tick()
+        let expectedMoveLeftAndTicksI = tetris.getCoordinates(`
+            --------
+            --------
+            --------
+            --------
+            ---XXXX-
+            --------
+        `)
+        assert.deepEqual(elementTetris.tetromino.coordinates, expectedMoveLeftAndTicksI)
+        stubForGetRandomTypeOfTetromino.returns("I")
+        elementTetris.tick()
+        let expectedFieldAddI = tetris.buildField(`
+            --------
+            --------
+            --------
+            --------
+            --X-----
+            XXXX-XXX
+        `)
+        assert.deepEqual(elementTetris.field.field, expectedFieldAddI)
+        let expectedNewTetrominoISecomd = tetris.getCoordinates(`
+            --XXXX--
+            --------
+            --------
+            --------
+            --------
+            --------
+        `)
+        assert.deepEqual(elementTetris.tetromino.coordinates, expectedNewTetrominoISecomd)
+        elementTetris.tick()
+        elementTetris.rotate()
+        elementTetris.moveRight()
+        elementTetris.moveRight()
+        elementTetris.tick()
+        let expectedTicksRotateMovesRightI = tetris.getCoordinates(`
+            --------
+            -----X--
+            -----X--
+            -----X--
+            -----X--
+            --------
+        `)
+        assert.deepEqual(elementTetris.tetromino.coordinates, expectedTicksRotateMovesRightI)
+        stubForGetRandomTypeOfTetromino.returns("O")
+        elementTetris.tick()
+        let expectedFieldAddISecond = tetris.buildField(`
+            --------
+            -----X--
+            -----X--
+            -----X--
+            --X--X--
+            XXXX-XXX
+        `)
+        assert.deepEqual(elementTetris.field.field, expectedFieldAddISecond)
+        let expectedNewTetrominoOSecond = tetris.getCoordinates(`
+            ---XX---
+            ---XX---
+            --------
+            --------
+            --------
+            --------
+        `)
+        assert.deepEqual(elementTetris.tetromino.coordinates, expectedNewTetrominoOSecond)
+        elementTetris.moveRight()
+        elementTetris.moveRight()
+        elementTetris.tick()
+        elementTetris.tick()
+        elementTetris.tick()
+        let expectedMovesRightTicksO = tetris.getCoordinates(`
+            --------
+            --------
+            --------
+            ---XX---
+            ---XX---
+            --------
+        `)
+        assert.deepEqual(elementTetris.tetromino.coordinates, expectedMovesRightTicksO)
+        stubForGetRandomTypeOfTetromino.returns("I")
+        elementTetris.tick()
+        let expectedFieldAddOSecond = tetris.buildField(`
```

Comment:
these variable names are meaningless. Check that every variable is actually reflecting what is going on, or you can just reassign single variable over and over in this test

Reply:
Still not fixed